README.md

```diff
@@ -142,3 +142,30 @@ an input and returns a str. By default, `RichHelpFormatter` sets the function to
 ```python
 RichHelpFormatter.group_name_formatter = str.title
 ```
+
```

Comment:
Thanks for the PR.
All of this should go in a `CONTRIBUTING.md` file so that GitHub suggests it when some one visits the Issues or Pull requests pages.